rotkehlchen/exchanges/bitstamp.py

```diff
@@ -269,7 +275,7 @@ def query_online_deposits_withdrawals(
         indices_to_delete = []
         for asset_movement in asset_movements:
             for idx, crypto_movement in enumerate(crypto_asset_movements):
-                if crypto_movement.timestamp == asset_movement.timestamp and crypto_movement.category == asset_movement.category and crypto_movement.asset == asset_movement.asset:  # noqa: E501
+                if crypto_movement.category == asset_movement.category and crypto_movement.asset == asset_movement.asset and crypto_movement.amount == asset_movement.amount + asset_movement.fee and abs(crypto_movement.timestamp - asset_movement.timestamp) < BITSTAMP_MATCHING_TOLERANCE:
```

Comment:
you will need the noqa: E501 due to the line legth. Also Do `<=`

```suggestion
                if crypto_movement.category == asset_movement.category and crypto_movement.asset == asset_movement.asset and crypto_movement.amount == asset_movement.amount + asset_movement.fee and abs(crypto_movement.timestamp - asset_movement.timestamp) <= BITSTAMP_MATCHING_TOLERANCE:  # noqa: E501
```

Reply:
Fixed